internal/battlemetrics/battlemetrics.go

```diff
@@ -38,10 +54,25 @@ func GetListOfRustServers(query ...string) RustServers {
 	for _, v := range query {
 		q += string(v)
 	}
-	search := SearchFilter + q
-	url := fmt.Sprintf("%v/servers%v%v%v%v", BattleMetricsURL, RustFilter, CountryFilter, PageFilter, search)
+	search := fmt.Sprintf("%v%v", SearchFilter, q)
+
+	urlString := fmt.Sprintf("%v/servers%v%v%v%v", BattleMetricsURL, RustFilter, CountryFilter, PageFilter, search)
+	url, err := url.Parse(urlString)
+	if err != nil {
+		log.Print(err)
+	}
 	fmt.Println(url)
-	res, err := http.Get(url)
+
+	client := http.Client{
+		Timeout: time.Second * 10,
+	}
+
+	req := &http.Request{
+		Method: "GET",
+		URL:    url,
+	}
```

Comment:
You already know this, but we can also build the request with the `http.NewRequest` function. That way we get an error to handle if there's an issue.